Sources/ApolloWebSocket/SplitNetworkTransport.swift

```diff
@@ -8,34 +8,22 @@ public class SplitNetworkTransport {
   private let webSocketNetworkTransport: NetworkTransport
   
   public var clientName: String {
-    get {
-      let httpName = self.httpNetworkTransport.clientName
-      let websocketName = self.webSocketNetworkTransport.clientName
-      if httpName == websocketName {
-        return httpName
-      } else {
-        return "SPLIT_HTTPNAME_\(httpName)_WEBSOCKETNAME_\(websocketName)"
-      }
-    }
-    set {
-      self.httpNetworkTransport.clientName = newValue
-      self.webSocketNetworkTransport.clientName = newValue
+    let httpName = self.httpNetworkTransport.clientName
+    let websocketName = self.webSocketNetworkTransport.clientName
+    if httpName == websocketName {
+      return httpName
+    } else {
+      return "SPLIT_HTTPNAME_\(httpName)_WEBSOCKETNAME_\(websocketName)"
     }
   }
 
   public var clientVersion: String {
-    get {
-      let httpVersion = self.httpNetworkTransport.clientVersion
-      let websocketVersion = self.webSocketNetworkTransport.clientVersion
-      if httpVersion == websocketVersion {
-        return httpVersion
-      } else {
-        return "SPLIT_HTTPVERSION_\(httpVersion)_WEBSOCKETNAME_\(websocketVersion)"
-      }
-    }
-    set {
-      self.httpNetworkTransport.clientVersion = newValue
-      self.webSocketNetworkTransport.clientVersion = newValue
+    let httpVersion = self.httpNetworkTransport.clientVersion
+    let websocketVersion = self.webSocketNetworkTransport.clientVersion
+    if httpVersion == websocketVersion {
+      return httpVersion
+    } else {
+      return "SPLIT_HTTPVERSION_\(httpVersion)_WEBSOCKETNAME_\(websocketVersion)"
```

Comment:
Hah, this is existing but this should probably be `WEBSOCKETVERSION`?

```suggestion
      return "SPLIT_HTTPVERSION_\(httpVersion)_WEBSOCKETVERSION_\(websocketVersion)"
```

Reply:
DERRRP good catch